### PR TITLE
Add system creation with config and props

### PR DESCRIPTION
**Pull request by chbatey**
You can do with with ActorSystemSetup but is a bit of boiler plate

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM

Out of curiosity, what's the use case, specifying dispatcher for the guardian or something?

File: akka-actor-typed/src/main/scala/akka/actor/typed/ActorSystem.scala [hunk 1]
```diff
@@ -219,8 +219,8 @@ object ActorSystem {
     apply(guardianBehavior, name, config)
 
   /**
-    * Java API: Create an ActorSystem
-    */
+   * Java API: Create an ActorSystem
+   */
```
Comment (patriknw) [inline akka-actor-typed/src/main/scala/akka/actor/typed/ActorSystem.scala]: if you are using "format on save" in IntelliJ you have to approve download of new version of scalafmt. It's a small notice in status bar (event log)

**Review by patriknw — APPROVED** [review]
LGTM, seems good to make it complete (same possibility as spawn)

**Review by raboof — APPROVED** [review]
Curious about the use case as well, but in any case nice for completeness

**Comment by patriknw** [comment]
Use case is to be able to set dispatcher, mailbox or actor tags for the user guardian actor in the same way as when spawning any other actor.

**Pull request merged**